Assets/Scripts/UI/MainMenu/MainMenuController.cs

```diff
@@ -23,6 +23,8 @@ public class MainMenuController : MonoBehaviour
     private GameObject defaultMenu;
     [SerializeField]
     private GalleryMenu galleryMenu;
+    [SerializeField]
+    private GameObject playerSelectModel;
```

Comment:
Only relevant part is the `PlayerSelectManager`, perhaps use that type here so that the intent is more explicit (and change the name)